link/src/main/java/com/stripe/android/link/ui/wallet/PaymentDetails.kt

```diff
@@ -0,0 +1,257 @@
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.RadioButton
+import androidx.compose.material.RadioButtonDefaults
+import androidx.compose.material.TabRowDefaults
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.MoreVert
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.stripe.android.link.R
+import com.stripe.android.link.theme.MinimumTouchTargetSize
+import com.stripe.android.link.theme.linkColors
+import com.stripe.android.link.theme.linkShapes
+import com.stripe.android.model.ConsumerPaymentDetails
+import com.stripe.android.model.ConsumerPaymentDetails.Card
+import com.stripe.android.R as StripeR
+
+@Composable
+internal fun PaymentDetailsListItem(
+    paymentDetails: ConsumerPaymentDetails.PaymentDetails,
+    enabled: Boolean,
+    isSelected: Boolean,
+    isUpdating: Boolean,
+    onClick: () -> Unit,
+    onMenuButtonClick: () -> Unit
+) {
+    Column {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .defaultMinSize(minHeight = 56.dp)
+                .clickable(enabled = enabled, onClick = onClick),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            RadioButton(
+                selected = isSelected,
+                onClick = null,
+                modifier = Modifier.padding(start = 20.dp, end = 6.dp),
+                colors = RadioButtonDefaults.colors(
+                    selectedColor = MaterialTheme.linkColors.actionLabelLight,
+                    unselectedColor = MaterialTheme.linkColors.disabledText
+                )
+            )
+            Column(
+                modifier = Modifier
+                    .padding(vertical = 8.dp)
+                    .weight(1f)
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    PaymentDetails(paymentDetails = paymentDetails)
+
+                    if (paymentDetails.isDefault) {
+                        DefaultTag()
+                    }
+
+                    val showWarning = (paymentDetails as? Card)?.isExpired ?: false
+                    if (showWarning && !isSelected) {
```

Comment:
why do we not show the warning if the PM is selected? That is kind of surprising -- presumably a user would still want to know if their card is expired _especially_ if they had selected that card

Reply:
Yh I see your point. I took out the isSelected check